test/e2e/app/app.go

```diff
@@ -101,6 +101,13 @@ type Config struct {
 
 	// Vote extension padding size, to simulate different vote extension sizes.
 	VoteExtensionSize uint `toml:"vote_extension_size"`
+
+	// Inject custom reactors (see node/main.go#startNode for a list of possibilities)
+	ExperimentalCustomReactors map[string]string `toml:"experimental_custom_reactors"`
+
+	// Experimental gossip "propagation rate" and "send once" features
```

Comment:
I believe this helps to understand that the feature is not available in the base reactor. The change is not that difficult when merging. 

Another approach would be to add a set of experimental parameters. My feeling is that it is a bit heavy weighted.

@cason was proposing also that instead of having experimental reactors under `fast-prototyping/reactors`, to have them in different branches. Such a solution is also fully do-able.